### PR TITLE
Enhancement tutorials

### PR DESCRIPTION
**Pull request by daniel-cyberbotics**
Suggestions of rewriting some parts to simplify or clarify the tutorials.

### CODE REVIEW

**Comment by daniel-cyberbotics** [comment]
:+1:

File: guide/tutorial-1-your-first-simulation-in-webots-20-minutes.md [hunk 1]
```diff
@@ -15,15 +15,20 @@ and with the basic concepts of Webots.
...
-suffix. The format of this file is derived from the **VRML** language, and is
+A **World** defines the initial state of a simulation.
+The different objects are called **Nodes** and are organized hierarchically
+in a **Scene Tree**. It means that a node can have some sub-nodes.
+
+<!-- -->
+
+> **Note**:
+A world is stored in a file having the ".wbt" extension.
+The file format is derived from the **VRML** language, and is
```
Comment (omichel) [inline guide/tutorial-1-your-first-simulation-in-webots-20-minutes.md]: Why did you remove the definition of **fields**?

File: guide/tutorial-1-your-first-simulation-in-webots-20-minutes.md [hunk 2]
```diff
@@ -39,48 +44,56 @@ stable.
...
+
 It's a good time to improve the scene light.
 
 > **Hands on**:
-Modify the following fields of the `DirectionalLight` node: `ambientIntensity`
-to 1, `direction` to [-0.33 -1 -0.5] and `castShadows` to TRUE.
+Modify the following fields of the `DirectionalLight` node:
+- `ambientIntensity` to 1 ;
+- `direction` to [-0.33 -1 -0.5] ;
+- `castShadows` to TRUE.
```
Comment (omichel) [inline guide/tutorial-1-your-first-simulation-in-webots-20-minutes.md]: **and** is missing.

File: guide/tutorial-1-your-first-simulation-in-webots-20-minutes.md (hunk 1 above)
Comment (daniel-cyberbotics) [inline guide/tutorial-1-your-first-simulation-in-webots-20-minutes.md]: I moved it just before using the 

> Each node has some customizable properties called fields. The first step is about to modify the background color.

**Comment by omichel** [comment]
The figure is very nice. However, I would a more precise terminology, e.g., instead of Node 1 and Node 2 (which are not referred to in the text), we could say *parent solid* and *solid end point*.

**Comment by fabienrohrer** [comment]
Small note for the future PRs, this is quite subjective, but the modifications on `guide/installation-procedure.md` seems to be out of the focus of this PR (in regard to the PR title, etc.). It would be better to keep 1 subject = 1 PR, even if a PR has 2 modified lines (simpler to comment, review, etc.).

**Comment by fabienrohrer** [comment]
The image `guide/images/hinge_joint.png` is precisely what I would like to see the `guide` (#178). From where does it come, did you create it?

**Comment by daniel-cyberbotics** [comment]
@fabienrohrer Yes, I took inspiration from [ODE wiki](http://ode-wiki.org/wiki/index.php?title=File:Hinge.jpg). As I didn't know if the image can be reused by us, I created one in Blender. Maybe I can put the .blend on the NAS ?

File: guide/installation-procedure.md [hunk 3]
```diff
@@ -47,11 +47,10 @@ The advantage of this solution is that Webots will be updated with the system
 updates. This installation requires the `root` privileges.
 
 First of all, you may want to configure your APT package manager by adding the Cyberbotics repository.
-You can easily add it from the `Software and Updates` application.
-In the `Other Software` tab, click on the `Add...` button and copy the following line:
+Simply execute the following line:
```
Comment (stefaniapedrazzi) [inline guide/installation-procedure.md]: Note that not all Webots users that work in Ubuntu are familiar with the terminal commands.
I'm not completely sure that it is good to remove the graphical procedure to add the Cyberbotics repository even if the terminal command is much simpler.

Comment (daniel-cyberbotics) [inline guide/installation-procedure.md]: Of course, but practically all steps are terminal commands here :)

**Comment by daniel-cyberbotics** [comment]
:+1:

**Comment by omichel** [comment]
I would use a transparent background to the image instead of setting an arbitrary grey.

**Comment by fabienrohrer** [comment]
> Maybe I can put the .blend on the NAS ?

Not necessarily, it was to be sure we have the rights for it. But I would be very happy if you could do the same for the other joints and to put them in the reference manual.

**Comment by daniel-cyberbotics** [comment]
:+1:

**Comment by omichel** [comment]
:+1:

**Comment by stefaniapedrazzi** [comment]
Just a small note about the milestone: a PR milestone should match a Webots release it not very useful it specifies the priority.
